x-pack/solutions/observability/plugins/observability/server/services/referenced_panel_manager.ts

```diff
@@ -45,6 +63,50 @@ export class ReferencedPanelManager {
     return panelId ? this.panelsById.get(panelId) : undefined;
   }
 
+  getPanelIndices(panel: DashboardPanel): Set<string> {
```

Comment:
Moved these here since `related_dashboards_client` was so large, and this slightly simplifies their impl.